appinfo/application.php

```diff
@@ -36,7 +37,7 @@ public function __construct(array $urlParams=array()){
 			['locking' => false]);
 
 
-		$container->registerService('HttpBindController', function($c){
+		$container->registerService('HttpBindController', function(IContainer $c) {
```

Comment:
Is this new pattern documented somewhere? It's quite hard to stay up to date as "part time" app developer with all changes in nc.